### PR TITLE
Update pokemon.json

### PR DESCRIPTION
**Pull request by Razikd29**

### CODE REVIEW

**Review by Aegyo — CHANGES_REQUESTED** [review]
Like I mentioned on discord, some of these changes weren't necessary (Unown variations, Megas) because of the fallback english translation redirecting to pokemon translation + alt key translation, but it is fine to leave it like this if you want. I've updated the translation instructions hopefully making this clearer.

I have commented on a couple of things you might have missed below, could you update your PR with those two things fixed please?

File: locales/fr/pokemon.json [hunk 1]
```diff
@@ -2,150 +2,155 @@
...
   "Venusaur": "Florizarre",
-  "Venusaur (Clone)": "",
-  "Mega Venusaur": "",
+  "Venusaur (Clone)": "Florizarre (Clone)",
+  "Spooky Venusaur": "Florizarre (Effrayant)",
+  "Venusaur (Rose)": "Florizarre (Fleur)",
+  "Mega Venusaur": "Méga-Florizarre",
   "Charmander": "Salamèche",
-  "Charmander (Clone)": "",
+  "Charmander (Clone)": "Salamèche (Clone",
```
[the remark this answers is not in the document]
Comment (Razikd29) [reply locales/fr/pokemon.json:45]: Well spotted, I'll correct with the rest

File: locales/fr/pokemon.json [hunk 2]
```diff
@@ -412,29 +418,29 @@
...
   "Mudkip": "Gobou",
   "Marshtomp": "Flobio",
   "Swampert": "Laggron",
-  "Mega Swampert": "",
+  "Mega Swampert": "Méga-Laggron",
   "Poochyena": "Medhyèna",
   "Mightyena": "Grahyèna",
   "Zigzagoon": "Zigzaton",
-  "Galarian Zigzagoon": "",
+  "Galarian Zigzagoon": "Zigzagoon de Galar",
```
[the remark this answers is not in the document]
Comment (Razikd29) [reply locales/fr/pokemon.json:441]: Oops, missed that one

**Comment by Razikd29** [comment]
I'll reverse the added lines and create an issue for the missing translation

**Comment by Aegyo** [comment]
You have quite a few ``[[alt.something]]` references which don't exist in the alt list at the top of the file, you can only use the ones that exist in there. I didn't include many alt forms there since they only exist once so making an extra variable for them didn't really seem needed - it would require the same amount of edits for translators.

Can you replace any non existent [[alt.blah]] references with just the translation for that variant?

The other updates look good though, thanks 👍

**Comment by Razikd29** [comment]
Sure, no problem 👍

**Review by Aegyo — APPROVED** [review]
All seems good now, thank you 👍

**Pull request merged**